cmd/ctr/app/main.go

```diff
@@ -119,11 +162,33 @@ containerd CLI
 		sandboxes.Command,
 		info.Command,
 	}, extraCmds...)
-	app.Before = func(context *cli.Context) error {
-		if context.GlobalBool("debug") {
+
+	var traceProvider *trace.TracerProvider
+
+	app.Before = func(cctx *cli.Context) error {
+		if cctx.GlobalBool("debug") {
 			logrus.SetLevel(logrus.DebugLevel)
 		}
+
+		trace := cctx.GlobalString("trace")
```

Comment:
nit: I would prefer the variable to be called `trace-endpoint`

Reply:
Will rename.

Reply:
Done.